giraph-block-app-8/src/main/java/org/apache/giraph/block_app/library/pagerank/PageRankInitializeAndNormalizeEdgesPiece.java

```diff
@@ -58,8 +54,6 @@
   public PageRankInitializeAndNormalizeEdgesPiece(
       ConsumerWithVertex<I, V, DoubleWritable, DoubleWritable> valueSetter,
```

Comment:
`valueSetter` should be completely removed, right?

Reply:
yes, only reason I didn't remove this is cause it would require constructor level changes and more changes in other files in general. But I can make them and test again